### PR TITLE
Styling header, admin, homepage + screening_support

### PR DESCRIPTION
**Pull request by noobsi**

### CODE REVIEW

**Comment by thaotonto** [comment]
ok

File: app/views/layouts/_user_header.html.erb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <li><%= link_to t("layouts.header.screening"), schedules_path %></li>
+      <li><%= link_to t("layouts.header.price"), price_path %></li>
+      <li><%= link_to t("layouts.header.about"), about_path %></li>
+      <li>
+        <div id="header-search" class="skip-content">
+          <div class="header-search-left">
+            <% if request.original_url.start_with? searchs_url %>
+               <%= form_tag searchs_path,
+                class: "form-inline", remote: true do %>
+                <%= render "searchs/search_form_content", keyword: keyword %>
```
Comment (ducbet) [inline app/views/layouts/_user_header.html.erb]: indent

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -10,9 +10,9 @@
   <body>
     <%= render "layouts/header" %>
     <div class="container">
-      <% flash.each do |message_type, message| %>
-        <%= content_tag :div, message, class: "alert alert-#{message_type}" %>
-      <% end %>
+      <dir id="flash_area">
+        <%= render "layouts/flash_message" %>
+      </dir>
```
Comment (ducbet) [inline app/views/layouts/application.html.erb]: div

**Comment by noobsi** [comment]
rubocop

**Comment by ducbet** [comment]
ok

**Pull request merged**